src/utils/Constants.ts

```diff
@@ -21,6 +21,8 @@ const IS_INDIVIDUAL_PAD = "IS_INDIVIDUAL_PAD"
 const MAX_MEMBERS_PAD = "MAX_MEMBERS_PAD"
 const EXISTING_PAYMENT_METHOD = "EXISTING_PAYMENT_METHOD"
 const EMPTY_AMOUNT = "EMPTY_AMOUNT"
+const EMPTY_FILTER = "EMPTY_FILTER"
+const EMPTY_VALUE = ""
```

Comment:
i think that we just need one EMPTY_VALUe you can remove the Empty filter we dont need it

Reply:
@imadiaz I have already made the requested changes